.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": "0.2.0",
```

Comment:
remove this or make it more generic for all the users.

Reply:
Yes, I removed it.